src/components/_internal/BaseTable/columns/actionsColumn.tsx

```diff
@@ -50,15 +50,19 @@ export const actionsColumn = {
   disableSortBy: true,
   cellType: CellTypes.actions,
   Cell: (props: CellProps<Record<string, unknown>>): React.ReactElement => {
-    const { row, rowActions } = props;
+    const { row, rowActions, rowActionsPaneWidth = 'auto' } = props;
     const actions = map((action) => ({
       ...action,
       onClick: () => action.onClick(row.id, row.original),
     }))(rowActions);
 
     return (
       <Inline justify="center" stretch="start">
-        <DropdownMenu actions={actions} paneWidth={140} placement="bottom-end">
+        <DropdownMenu
+          actions={actions}
+          paneWidth={rowActionsPaneWidth}
```

Comment:
This seems to be overly complicated. I was thinking about setting the `paneWidth="auto"` and letting the content decide on how wide the menu should be. It should also limit the number of errors in tests :)

Reply:
that's the default value, 'auto' and it's auto-adjusting based on the content

Reply:
Yeah I know I don't want to pollute Datatable component with another property if we can do the sizing automatically. I think that DT is already too complicated